doc.md

```diff
@@ -15,20 +15,21 @@
 </script>
 
 3Dmol.js is an object-oriented, WebGL based JavaScript library for online molecular visualization - No Java required!
-With 3Dmol.js, you can add beautifully rendered molecular visualizations to your web applications. Features include:
+With 3Dmol.js, you can add beautifully rendered molecular visualizations to your web applications. 
 
-* support for pdb, sdf, mol2, xyz, and cube formats
-* parallelized molecular surface computation
-* sphere, stick, line, cross, cartoon, and surface styles
-* atom property based selection and styling
-* labels
-* clickable interactivity with molecular data
-* geometric shapes including spheres and arrows
+Features include:
+
+1. Support for pdb, sdf, mol2, xyz, and cube formats
```

Comment:
This should not be an ordered list, it is unordered.

Reply:
The theme is not supporting unordered lists as of now, I have asked the maintainer of the same for more info

Reply:
This is fixed, hence resolving this :)